charts/caraml/README.md

```diff
@@ -0,0 +1,26 @@
+# caraml
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.16.0](https://img.shields.io/badge/AppVersion-1.16.0-informational?style=flat-square)
+
+A Helm chart for Kubernetes
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| caraml-dev | <caraml-dev@caraml.com> |  |
+
+## Values
+
+| Key | Type | Default | Description |
```

Comment:
let's cleanup this since don't have this dependencies anyway

Reply:
removed sample values, and updated docs